### PR TITLE
docs: highlight scripts in markdown

### PR DESCRIPTION
**Pull request by nouraellm**
This PR highlights all lines of code/cmds/error messages by specifying the used language

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/BAZEL.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ Usually there is a single item (or multiple items of the same kind) where the ov
 #### bazel run
 If you see the following error:
 
-```
+```sh
```
Comment (bob-watson) [inline docs/BAZEL.md:264]: This block looks like output and should probably not be formatted as a shell command.

File: docs/BAZEL.md [hunk 2]
```diff
@@ -282,16 +282,14 @@ e.g: `yarn bazel test packages/core/test/bundling/forms:symbol_test`
 
 #### mkdir missing
 If you see the following error::
-```
-
+```sh
```
Comment (bob-watson) [inline docs/BAZEL.md:285]: This block looks like output and should probably not be formatted as a shell command.

File: docs/PUBLIC_API.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ Using yarn ensures that you are running the correct version of Bazel.
 
 Here is an example of a Circle CI test failure that resulted from adding a new allowed type to a public property in `core.d.ts`. Error messages from the API guard use [`git-diff` formatting](https://git-scm.com/docs/git-diff#_combined_diff_format).
 
-```
+```sh
```
Comment (bob-watson) [inline docs/PUBLIC_API.md:70]: This block looks like output and should probably not be formatted as a shell command.

File: docs/BAZEL.md [hunk 4]
```diff
@@ -303,7 +301,7 @@ After that, a `git clean -xfd`, `yarn`, and `yarn build` should resolve this iss
 
 If you see the following error:
```
Comment (bob-watson) [inline docs/BAZEL.md:302]: Move the command from the following block with the error messages to here, such as: 
```suggestion
If running `yarn bazel build packages/...` returns the following error:
```

File: docs/BAZEL.md [hunk 5]
```diff
@@ -303,7 +301,7 @@ After that, a `git clean -xfd`, `yarn`, and `yarn build` should resolve this iss
...
 
-```
+```sh
 $ yarn bazel build packages/...
```
Comment (bob-watson) [inline docs/BAZEL.md:305]: It might also be more clear in this context to move the command to before this block.

File: docs/BAZEL.md [hunk 6]
```diff
@@ -303,7 +301,7 @@ After that, a `git clean -xfd`, `yarn`, and `yarn build` should resolve this iss
 
 If you see the following error:
 
-```
+```sh
```
Comment (bob-watson) [inline docs/BAZEL.md:304]: This block is mostly output and should probably not be formatted as a shell command.

**Review by bob-watson — CHANGES_REQUESTED** [review]
Thanks for your revisions!
I added a couple of comments where output was formatted as shell, which can cause some inappropriate formatting when keywords are present in the output.

**Comment by nouraellm** [comment]
@bob-watson thank you very much for the review, changes requested have been applied as required!
Everyone else who got the notification to review this PR .. sorry 🙈 fixup commits are still new to me.
Good day!

**Review by bob-watson — APPROVED** [review]
LGTM

Reviewed-for: global-docs-approvers

**Review by jessicajaniuk — APPROVED** [review]

**Review by atscott — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 1cb93ce4e17e1a04e74303c0f4b89c2019e34337.

**Pull request closed without merging**